.github/actions/npm-rc-release/action.yml

```diff
@@ -0,0 +1,46 @@
+name: 'npm rc release'
+description: 'Publish code changes to npmjs.org'
+inputs:
+  git_email:
+    description: 'Email of git commit containing version bump'
+    required: true
+  git_username:
+    description: 'Commit author name'
+    required: true
+  npm_token:
+    description: 'npm token for publishing. Login to npmjs.org to generate this.'
+    required: true
+outputs:
+  release_tag:
+    description: 'The name of the created git tag'
+    value: ${{ steps.release-tag-step.outputs.release-tag }}
+runs:
+  using: 'composite'
+  steps:
+    - name: 'Configure git CLI'
+      shell: bash
+      run: |
+        git config --global user.email "${{ inputs.git_email }}"
+        git config --global user.name "${{ inputs.git_username }}"
+    - name: 'Bump version'
+      shell: bash
+      run: npm version prerelease --preid=rc --no-git-tag-version
+    - name: 'Commit and tag version changes'
+      shell: bash
+      run: |
+        git add .
+        newVersion=$(jq -r '.version' package.json)
+        git commit -m "[skip ci] Publish release v$newVersion"
+        git tag "v$newVersion"
+    - name: 'Output release tag'
+      shell: bash
+      id: release-tag-step
+      run: echo "release-tag=$(git describe --tags)" >> $GITHUB_OUTPUT
+    - name: 'Push changes'
+      shell: bash
+      run: git push --follow-tags
```

Comment:
I think we should remove any git tagging we're doing here for release candidates. We can probably emit the tag we used to publish in this step if we want to echo it out in the workflow (`run: echo "${{ steps.npm-rc-release.outputs.release_tag }} was published"`), but it's up to you. 🤔

Reply:
Okay, will do!

Reply:
(I think if we remove these tagging parts, the reference to `...outputs.release_tag` won't work anymore, but maybe you don't mean to remove all the refs to `release_tag`? LMK!)